pybossa/view/projects.py

```diff
@@ -1502,9 +1502,11 @@ def tasks_browse(short_name, page=1, records_per_page=None):
     try:
         args = parse_tasks_browse_args(request.args)
         view_type = request.args.get('view')
+        task_browse_default_page_size = 10
+        task_list_default_page_size = 30
         if view_type != 'tasklist' and (current_user.subadmin or current_user.admin or current_user.id in project.owners_ids):
             # owners and (sub)admin have full access, default size page for owner view is 10
-            per_page = records_per_page if records_per_page in allowed_records_per_page else 10
+            per_page = records_per_page if records_per_page in allowed_records_per_page else task_browse_default_page_size
```

Comment:
Looks like it it still possible that per_page can be set to 100. Should [this](https://github.com/bloomberg/pybossa/pull/694/files#diff-7fa4bf701959a15b1589099598a22b2352b452e47a90c7b981d5d340d8bc45dfR1492) be changed so that max per_page would be 50 instead of 100?

Reply:
From the discussion, I thought we are going to change the task-list default per_page value from 100 to 30, while still allow users to explicitly select 100 per_page. no?

Reply:
Let's confirm this offline w/ team and proceed accordingly. Thanks!